packages/funding-rate-proposer/index.js

```diff
@@ -117,7 +117,7 @@ async function run({
           message: "End of serverless execution loop - terminating process"
         });
         await waitForLogger(logger);
-        await delay(4); // waitForLogger does not always work 100% correctly in serverless. add a delay to ensure logs are captured upstream.
+        await delay(2); // waitForLogger does not always work 100% correctly in serverless. add a delay to ensure logs are captured upstream.
```

Comment:
Reverting [last commit](https://github.com/UMAprotocol/protocol/commit/eb55ff2941bd5fb587c5db992a6bd8a89672b626) that didn't patch the bug.